app/api/CardanoClientApi.js

```diff
@@ -40,6 +40,7 @@ export default class CardanoClientApi {
   async getWallets() {
     console.debug('CardanoClientApi::getWallets called');
     const response = await ClientApi.getWallets();
+    response.shift(); // TODO: Remove this before merging
```

Comment:
This needs to be removed before we merge, backend is still returning default wallet so this has been introduced only for testing purpuses.